python/py-astunparse/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-astunparse
+version             1.6.3
+revision            0
+
+license             BSD
+maintainers         nomaintainer
+platforms           darwin
+supported_archs     noarch
+
+description         An AST unparser for Python.
+
+long_description    ${description}. This is a factored out version\
+                    of unparse found in the Python source distribution\;\
+                    under Demo/parser in Python 2 and under Tools/parser\
+                    in Python 3.
+
+homepage            https://github.com/simonpercivall/astunparse
+
+checksums           rmd160  3506f9ca63bfac6efa0a2933ff0257b760972011 \
+                    sha256  5ad93a8456f0d084c3456d059fd9a92cce667963232cbf763eac3bc5b7940872 \
+                    size    18290
+
+supported_archs     noarch
+                    
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    post-destroot {
```

Comment:
please move the `post-destroot` phase after the test.run ; as is conventionally done

Reply:
Thanks.